pkg/agent/controller/traceflow/packetin_test.go

```diff
@@ -211,7 +202,7 @@ func TestParseCapturedPacket(t *testing.T) {
 	}
 }
 
-func getTestPacketBytes() []byte {
+func getTestPacketBytes(dstIPv4 string) []byte {
```

Comment:
Maybe rename it as `dstIP`, it's confusing to see a global variable and a parameter with the same name.